src/main/java/pl/umk/mat/git2befit/service/workout/TrainingPlanService.java

```diff
@@ -54,6 +55,8 @@ public TrainingPlanService(
             return ResponseEntity.badRequest().header("Cause", "wrong date format").build();
         } catch (IllegalArgumentException e) {
             return ResponseEntity.status(HttpStatus.CONFLICT).header("Cause", e.getMessage()).build();
+        } catch (EquipmentCountException e) {
+            return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
```

Comment:
Jak dobrze pamiętam to mieliśmy chyba w causach dawać jakieś info odnośnie tego co się rozwaliło

Reply:
Done